wdio.conf.js

```diff
@@ -20,6 +20,14 @@ if (process.env.npm_package_name !== 'terra-framework') {
     path.join(__dirname, 'packages', 'terra-brand-footer', 'tests', 'wdio', '**', '*-spec.js'),
     path.join(__dirname, 'packages', 'terra-collapsible-menu-view', 'tests', 'wdio', '**', '*-spec.js'),
     path.join(__dirname, 'packages', 'terra-date-input', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-date-picker', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-date-time-picker', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-dialog-modal', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-embedded-content-consumer', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-hookshot', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-infinite-list', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-layout', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-menu', 'tests', 'wdio', '**', '*-spec.js'),
```

Comment:
This is to prevent wdio tests in all the other packages from running. This will be reverted in the [last PR (Part 3/3)](https://github.com/cerner/terra-framework/pull/1368/files#diff-f297ad33212595210fbfc7c32d334f7025fe8eee14cc4179d2527c3f4543035f).